app/i18n/index.js

```diff
@@ -1,36 +1,35 @@
-import i18n from 'i18next';
-import { initReactI18next } from 'react-i18next';
-import moment from 'moment';
+import i18n from "i18next";
```

Comment:
Please do t change the existing style

Reply:
I am so sorry, wasn't intentional. I was working with neovim and something about how it deals with copy/paste or something else entirely might have caused this, let me fix it.